crates/nu-plugin-engine/src/process.rs

```diff
@@ -79,7 +79,7 @@ impl PluginProcess {
         }
     }
 
-    /// Move the plugin process out of the foreground. See [`ForegroundGuard::reset`].
```

Comment:
Does this member still exist? Maybe it has the more specific doccomment itself.

Reply:
Doesn't exist from what I see. There is `reset_internal` though, but it only says it must be called only once within `drop`

Reply:
OK, then it's probably fine to just point to the `ForegroundGuard` around the meaning of foreground.

Reply:
done